lib/shopify_app/sessions_controller.rb

```diff
@@ -20,6 +20,7 @@ def callback
         session[:shopify_domain] = shop_name
 
         WebhooksManager.queue(shop_name, token) if ShopifyApp.configuration.has_webhooks?
+        ScripttagsManager.queue(shop_name, token) if ShopifyApp.configuration.has_scripttags?
```

Comment:
we could actually break these into a method here and then that method could be overridden (with a super) to allow custom jobs to be queued from this point as well. Essentially the same thing I have on shopify-sinatra-app an after_auth_callback.

Lets not make that change in this PR, I'm just pointing it out for the discussion in #209